runescape-client/src/main/java/class171.java

```diff
@@ -4,20 +4,22 @@
 import net.runelite.mapping.ObfuscatedSignature;
 
 @ObfuscatedName("fd")
-public class class171 implements class170 {
+public class class171 implements class170
+{
```

Comment:
This bracket is supposed to go next to the class declaration but it doesn't really matter thát much